ui/lib/nav.ts

```diff
@@ -41,6 +42,24 @@ export const getParentNavValue = (
   }
 };
 
+export const getPageLabel = (currentPage: string): string => {
+  const parsed = qs.parse(useLocation().search);
```

Comment:
:thinking:  Not sure how this is working. I was under the impression that you cannot call hooks from plain old JS like this. I thought it had to be within another hook.

Reply:
Me too but it works so I'm not asking questions